smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -116,7 +116,7 @@ public async Task UpdatePackageState()
                     if (this.overwritePackages)
                     {
                         Console.WriteLine($"{package}: found. Removing package.");
-                        await Process.RunAsync("apt", $"purge -y {package}");
+                        await Process.RunAsync("apt", $"purge -y {package}", 180);
```

Comment:
This will allow the tests time to successfully uninstall the latest run of iotedge.